test/main/errors/index.test.js

```diff
@@ -0,0 +1,105 @@
+import * as Sentry from '@sentry/electron'
+import { init } from '../../../main/errors'
+
+jest.mock('@sentry/electron', () => ({ init: jest.fn(), IPCMode: { Classic: 'test-ipcmode' } }))
+jest.mock('../../../main/store')
+
+beforeAll(() => {
+  jest.useFakeTimers()
+})
+
+afterAll(() => {
+  jest.useRealTimers()
+})
+
+describe('sentry', () => {
+  const mockEvent = (event) => Sentry.init.mock.calls[0][0].beforeSend(event)
+  const mockEvents = (events) => events.map(mockEvent)
+  const validEvent = {
+    exception: {
+      values: [],
+    },
+    extra: {
+      networks: '{}',
+      networksMeta: '{}',
+      tokens: '{}',
+    },
+    tags: {
+      "frame.instance_id": undefined,
+    },
+    user: {
+      ip_address: undefined,
+    }
+  }
+
+  it('should initialize sentry with the expected object', () => {
+    init()
+    expect(Sentry.init).toHaveBeenCalledWith({
+      beforeSend: expect.any(Function),
+      dsn: 'https://7b09a85b26924609bef5882387e2c4dc@o1204372.ingest.sentry.io/6331069',
+      ipcMode: 'test-ipcmode'
+    })
+  })
+
+  it('should strip asar paths from stackframe modules', () => {
```

Comment:
@goosewobbler can we add a test for non-Windows style paths here just to be safe?